### PR TITLE
Add notebook for training relevance model

### LINKED ISSUE #141: Fine tune relevance pre-trained model
As a part of setting up the training pipeline in issue #11, we'd have to write a notebook that fine tunes a model to find relevant paragraphs to a kpi question. Also report performance metrics for the trained model. 

Acceptance
- [ ] Add a train-relevance.ipynb notebook

### PR DESCRIPTION
**Pull request by chauhankaranraj**
This PR adds a notebook (updated version of `train_infer_pipeline.ipynb`) which trains (well, fine-tunes, to be more precise) and evaluates the relevance model on the ~150 pdf samples provided by Christian and team.

Additionally, it makes some minor changes to path configs defined for model training and evaluation.

Closes #141 
Related to #11

### CODE REVIEW

**Comment by chauhankaranraj** [comment]
Fixed the merge conflicts. Could please take a look and review the notebook?
/cc @MichaelClifford @erikerlandson

File: notebooks/demo2/train_relevance.ipynb [hunk 1]
```diff
@@ -0,0 +1,5502 @@
+{
```
Comment (oindrillac) [inline notebooks/demo2/train_relevance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.        s3_key="relevance_fine_tune_demo_scores,csv",
</pre><p>should this be <code>.csv</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (chauhankaranraj) [reply notebooks/demo2/train_relevance.ipynb:1]: Ah yes, it should be :)

Comment (chauhankaranraj) [reply notebooks/demo2/train_relevance.ipynb:1]: Updated!

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Pull request merged**